botocore/data/emr/2009-03-31/waiters-2.json

```diff
@@ -37,6 +37,25 @@
           "expected": "TERMINATED_WITH_ERRORS"
         }
       ]
+    },
+    "ClusterTerminated": {
+      "delay": 30,
+      "operation": "DescribeCluster",
+      "maxAttempts": 60,
+      "acceptors": [
+        {
+          "state": "success",
+          "matcher": "path",
+          "argument": "Cluster.Status.State",
+          "expected": "TERMINATED"
+        },
+        {
+          "state": "failure",
```

Comment:
@jamesls I am not sure what you are thoughts about this but it seemed interesting so figured to point this out. So I agree it is a good thing to raise any detected errors to the user, but the purpose of this waiter is to wait until terminated. So should we be raising an error if we reached the ``TERMINATED_WITH_ERRORS`` as the cluster did terminate? Honestly, I can see users wanting both behaviors with one side not caring that there were errors and another wanting to know if there were errors when terminating. I think it is fine with this approach, but I can see another waiter added that differentiates between the two.